.github/workflows/wheels-build.yaml

```
@@ -19,7 +19,7 @@ on:
       script:
         required: true
         type: string
-      wheel_type:
+      package_type:
```

```suggestion
      package-type:
```

Could we use `package-type`, to make it exactly match the `wheels-publish` workflow?

https://github.com/rapidsai/shared-workflows/blob/51e5d508a8704ff11227af71d276e5b7c4d60d6b/.github/workflows/wheels-publish.yaml#L24-L28

I know it seems like a small thing, but being consistent here makes it clearer that these 2 inputs really do refer to the exact same concept, with the exact same set of valid values.

---

There's an input called `build_type` within this file, and seems to be a bit problematic to have a hypenated input along with an underscored input in the file. Let me know how do you want to resolve this.

---

That difference in casing already exists throughout these workflows.

For example:

https://github.com/rapidsai/shared-workflows/blob/51e5d508a8704ff11227af71d276e5b7c4d60d6b/.github/workflows/wheels-test.yaml#L27-L33

I wouldn't worry about it. I think that's less problematic than having two inputs whose names differ **only** by case.